### PR TITLE
Wip/gtkmm 1

### PR DESCRIPTION
**Pull request by guruDanny67**
The full gtkmm stack!

### CODE REVIEW

File: gvsbuild/projects.py [hunk 1]
```diff
@@ -1841,3 +1847,133 @@ def build(self):
...
+            archive_url = 'https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.0.tar.xz',
+            hash = 'f843d6346260bfcb4426259e314512b99e296e8ca241d771d21ac64f28298d81',
+            dependencies = [
+                'meson',
+                'ninja',
+            ]
+        )
+
+    def build(self):
+        Meson.build(self)
```
Comment (nacho) [inline gvsbuild/projects.py]: does it have a license?

Comment (nacho) [inline gvsbuild/projects.py]: does it have a license?

Comment (guruDanny67) [reply gvsbuild/projects.py]: Forgotten ;(

Updated with the license and, while I copy it, correct also the name, libsigc++

**Comment by guruDanny67** [comment]
I see there are some problem with the x64 build, one on libsigc++ (that I fixed) and one on gtkmm (that's a little bit complicated, some unresolved external) so until is ok also on x64 the merge is blocked :(

But I'm using the PR to test with appveyor :)

**Comment by acidtonic** [comment]
Would be interested in seeing this completed. I had to give up on gvsbuild because I could not get gtkmm to build using the provided tools and it doesn't supply it.

Sadly now using vcpkg because gtkmm is supported without fighting the compile on windows.... (perl required, XML::Parser required, missing glibmmconfig.h file even though my build works it never generates it... etc.)

**Pull request closed without merging**